templates/resources/includes/xl-deploy-run-script.j2

```diff
@@ -0,0 +1,61 @@
+# Copy default plugins
```

Comment:
Shouldn't the xl-deploy and xl-release snippets be stored in the `xl-deploy` and `xl-release` directories? For example, `resources/xl-deploy/run-in-container-suffix.sh.j2`?

Reply:
Would be nicer... will take a look